srcpkgs/tuptime/srcpkgs/tuptime/REMOVE

```diff
@@ -0,0 +1,6 @@
+# Remove tuptime db file
+case "${ACTION}" in
+pre)
+        rm -f var/lib/tuptime/tuptime.db
```

Comment:
This should be in `mutable_files`, I think, and not removed automatically on uninstall.

Reply:
Ok, added `mutable_files` and removed the REMOVE file